operations/iuf/workflows/product_delivery.md

```diff
@@ -64,6 +64,7 @@ Refer to that table and any corresponding product documents before continuing to
 
      **`NOTE`** When installing USS 1.1 or higher, select either SLURM or PBS Pro Products to use on the system before running this stage. For more information, see the `deliver-product` stage
      details in the "Install and Upgrade Framework" section of the _HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray Supercomputing EX Systems (S-8063)_.
+      For creating site_vars.yaml in admin directory, refer to [Populate admin directory with files defining site preferences](configuration.md#1-populate-admin-directory-with-files-defining-site-preferences).
```

Comment:
This is an important step in the whole process. I think the whole "deliver_product.md" page should have a new step that populates the admin directory. I think this could be the new step 3 on the page and then "Execute the IUF deliver-product stage" could become step 4.

Reply:
Thanks @leliasen-hpe  , Added as a sub-section 3